srcpkgs/lib3mf/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'lib3mf'
+pkgname=lib3mf
+version=2.2.0
+revision=1
+build_style=cmake
+configure_args="-DUSE_INCLUDED_LIBZIP=OFF -DUSE_INCLUDED_ZLIB=OFF
+ -DUSE_INCLUDED_GTEST=OFF -DUSE_INCLUDED_SSL=OFF
+ -DCMAKE_INSTALL_INCLUDEDIR='include/$pkgname' -DCMAKE_INSTALL_PREFIX=/usr
+ $([ "$XBPS_CHECK_PKGS" ] || echo '-DLIB3MF_TESTS=OFF')"
+hostmakedepends="pkg-config"
+makedepends="zlib-devel libzip-devel"
+checkdepends="gtest-devel openssl-devel"
+short_desc="Official 3mf format implementation"
+maintainer="Danielle Hutzley <endergeryt@gmail.com>"
+license="BSD-2-Clause"
+homepage="https://3mf.io"
+distfiles="https://github.com/3MFConsortium/lib3mf/archive/refs/tags/v${version}.tar.gz"
+checksum=96e85e278fc5474123e3c47237dd42faaf1fdf8e182541a84af7fe84ddd3cbde
+
+post_install() {
+	vdoc README.md
+	vlicense LICENSE
+}
+
+lib3mf-devel_package() {
+	depends="${sourcepkg}>=${version}_${revision} $makedepends"
+	short_desc+=" - development files"
+	pkg_install() {
+		INCDIR="$DESTDIR/usr/include/lib3mf/"
+		BINDDIR="$INCDIR/Bindings"
+		ln -rst "$INCDIR" "$BINDDIR"/C/* "$BINDDIR"/Cpp/*
```

Comment:
why do this instead of just `mv ${DESTDIR}/Bindings/C/* ${DESTDIR}/Bindings/Cpp/* ${DESTDIR}/usr/include/$pkgname`?

also, this would fit more in `post_install()`

Reply:
Both formats might be used, so it's best to just use symlinks in case the original locations are needed.

Reply:
Putting it in `post_install` didn't work, so I used `pre_install` instead. Seems to have fixed it.